fem/gridfunc.hpp

```diff
@@ -640,17 +735,35 @@ class GridFunction : public Vector
                                        ) const
    { ComputeElementLpErrors(infinity(), exsol, error, NULL, irs); }
 
-   /** When given a vector weight, compute the pointwise (scalar) error as the
-       dot product of the vector error with the vector weight. Otherwise, the
-       scalar error is the l_2 norm of the vector error. */
+   /// @brief Returns ||u_ex - u_h||_Lp for vector fields
+   ///
+   /// When given a vector weight, compute the pointwise (scalar) error as the
```

Comment:
This is an improvement, but if we are improving documentation, it would be good to explain all the arguments. In particular, the 2 coefficients `weight` and `v_weight` could be explained.

Reply:
Good point. I added parameter documentation and a couple explanatory equations. I suspect this PR is now mostly documentation :bowtie:

Reply:
Thanks, the documentation is greatly improved now.